mozregression/pyinstaller.py

```diff
@@ -0,0 +1,136 @@
+"""PyInstaller helper module to enable packaging tcl/tk with app bundles."""
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2005-2023, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+# ----------------------------------------------------------------------------
```

Comment:
I think this needs to be GPL-only, unless I misunderstand the licensing exceptions.

Reply:
I think you are right here. I'll have to check how this can be released as both Mozilla and PyInstaller GPL within a project that uses MPL. The copied parts are fairly limited (mainly lines 96 - 104, and some of `resign_binary`. I may be able to put those in a separate file altogether and keep this file MPL. This might be a better approach.